docker-compose.yml

```diff
@@ -0,0 +1,39 @@
+version: '3'
+networks:
+  db-data-net:
+    driver: bridge
+services:
+  pythonapp:
+    image: todoapp:3.0.0
+    build:
+      context: .
+      dockerfile: Dockerfile
+    container_name: todoapp
+    ports:
+      - "8081:8080"
```

Comment:
What is the reason why you don't use 8080 port in the local machine? If some container is running on it you need to stop and delete it